bin/configure.js

```diff
@@ -74,12 +74,15 @@ function requirements() {
   var required = $.semver('1.7.0');
   $.childProcess.exec('java -version', function(error, stdout, stderr) {
     if (error || !stderr) {
-      $.util.error([
-            'Unable to get java version.',
-            'Please install java before building.',
-          ].join('\n'));
+      console.error([
```

Comment:
Seems like this file should be reverted?

Reply:
cf below, updated this this API is deprecated.